ui/src/app/views/workspace/component.ts

```diff
@@ -0,0 +1,63 @@
+import {Component, OnInit} from '@angular/core';
+import {Router} from '@angular/router';
+
+import {Repository} from 'app/models/repository';
+import {RepositoryService} from 'app/services/repository.service';
+import {User} from 'app/models/user';
+import {UserService} from 'app/services/user.service';
+
+class Cohort {
```

Comment:
Once #21 is in (soon I think?), you should be able to pick up `Cohort` and `CohortsService` from there.